index.html

```diff
@@ -892,11 +860,42 @@ <h4>Create Proof (eddsa-jcs-2022)</h4>
 is produced as output.
             </p>
 
-            <p>
-The algorithm is the same as the one described in Section
-[[[#create-proof-eddsa-rdfc-2022]]] with the changes outlined in
-Section [[[#eddsa-jcs-2022]]].
-            </p>
+            <ol class="algorithm">
+              <li>
+Let |proof| be a clone of the proof options, |options|.
+              </li>
+              <li>
+Set <var>proof</var>.<var>@context</var> to
+<var>unsecuredDocument</var>.<var>@context</var>.
+              </li>
+              <li>
+Let |proofConfig| be the result of running the algorithm in
+Section [[[#proof-configuration-eddsa-jcs-2022]]] with
+|proof| passed as a parameter.
```

Comment:
This might be a bit more clear:

```suggestion
|proof| passed as the <em>proof options</em> parameter.
```